App/my-app/src/App.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import './App.css';
+import { render } from '@testing-library/react';
+import { Item } from './components/Item';
+import Header from './components/Header';
+import CartItem from './components/CartItem';
+
+export default class App extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.incrementQuantity = this.incrementQuantity.bind(this);
+    this.state = {
+      headerList: [],
+      itemList: [],
+      cartList: [],
+      loader: false,
+      quantity: 0
+    }
+  }
+
+  incrementQuantity = () => {
+    this.setState(ps => {
+      return { quantity: ps.quantity + 1 }
+    })
+  }
+
+  componentDidMount() {
+    setTimeout(() => {
+      this.setState({
+        headerList: [{ title: "Unicorn Land", subtitle: "Kids fav place" }],
+        itemList: [
+          { id: 1, name: "Doll", recommendedAge: 5, recommendedGender: "girl", price: 30 },
+          { id: 2, name: "Train", recommendedAge: 4, recommendedGender: "boy", price: 12 },
+          { id: 3, name: "Ball", recommendedAge: 6, price: 30 },
+          { id: 4, name: "Puzzle", recommendedAge: 12, price: 30 },
+          { id: 5, name: "Teddy bear", recommendedAge: 3, price: 30 },
+          { id: 6, name: "Kite", recommendedAge: 10, price: 30 }
+        ],
+        loader: true
+      })
+    }, 3000);
+  }
+
+  renderHeader = () => { return this.state.headerList.map(headerItem => <Header title={headerItem.title} subtitle={headerItem.subtitle} />) }
```

Comment:
Ai un singur element de header, un obiect. Nu e nevoie sa il faci array, sa il parcurgi, si sa randezi. 
E suficient daca functia ta va returna direct <Header title={this.state.header.title} ........ />